spec/support/screening_lists_search_behavior.rb

```diff
@@ -586,18 +412,6 @@
   it_behaves_like 'it contains all expected results of source'
 end
 
-shared_examples 'it contains all ScreeningList::Ssi results that match countries "RU"' do
-  let(:source) { ScreeningList::Ssi }
-  let(:expected) { (0..3).to_a }
-  it_behaves_like 'it contains all expected results of source'
-end
-
-shared_examples 'it contains all ScreeningList::Ssi results that match countries "UA,DJ"' do
-  let(:source) { ScreeningList::Ssi }
-  let(:expected) { [] }
-  it_behaves_like 'it contains all expected results of source'
-end
-
```

Comment:
After reviewing the consolidated spec more closely, I can see that coverage of the various query params is already sufficient. So I'm going to merge this as it is now.